src/utils.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @module utils */
+/**
+ * @fileoverview Contains generic utility functions used throught the code.
+ */
+
+/**
+ * Generates and returns a random number(integer/float) within the limits set.
+ * @param {(number|undefined)} low Minimum Limit
+ * @param {(number|undefined)} high Maximum Limit
+ * @param {object} [options = {}] Optional Additional parameters.
+ * @param {boolean=} options.returnInteger Returns an integer.
+ * @return {number} Random Number.
+ */
+function getRandomNumber(low, high, options = {}) {
+  low = low || Number.MIN_SAFE_INTEGER;
+  high = high || Number.MAX_SAFE_INTEGER;
+  let result = Math.random() * (high - low) + low;
```

Comment:
What happens one of min, max value is not set?
Given the wide range of JS Max/Min safe integer, do you think it makes sense to use that or have a strict validation for args?

Reply:
@siriscac 
1) For primitive values like integer, number its not a problem to have a big number.
2) For use cases like generating lengthOfMockArray, lengthOfString,  we send the low and high set by us if the oasDoc doesn't have any constraints set.

`
const low = schema.minLength || 1;

const high = schema.maxLength || low + 10;

const lengthOfString = getRandomNumber(low, high, {returnInteger: true});

return getRandomString(lengthOfString);
`
This helps to escape from generating a very large number, or a negative number.
We chose to do it this way because schema.minLength, schema.maxLength are optional fields, and we need to generate strings for mock requestBody even if they don't have these optional fields.

Reply:
LGTM